README.md

```diff
@@ -17,72 +17,73 @@ $ export PATH=$HOME/bin:$PATH
 $ export CAPSTAN_QEMU_PATH=`which qemu-system-x86_64` 
 ```
 
-## Build
+## Building the application
 
-The ``build`` command create image.  If you make changes to the application, 
-you need rebuild the image with:
+The ``build`` command creates a Capstan virtual machine image.
```

Comment:
Capstan is the tool to build the virtual machine and the OSv disk image. The output disk image format depends on the hypervisor selected.

The ``build`` command creates a virtual machine running on Virtual Box.